server/db/migrations/20190611202308-create-borrowed-books.js

```diff
@@ -1,41 +1,48 @@
 /* eslint-disable require-jsdoc */
-export function up(queryInterface, Sequelize)  {
-    return queryInterface.createTable('BorrowedBooks', {
-      id: {
-        allowNull: false,
-        autoIncrement: true,
-        primaryKey: true,
-        type: Sequelize.INTEGER
-      },
-      isbn: {
-        allowNull: false,
-        type: Sequelize.INTEGER,
-      },
-      title: {
-        allowNull: false,
-        type: Sequelize.STRING
-      },
-      dateBorrowed: {
-        allowNull: false,
-        type: Sequelize.DATE
-      },
-      dueDate: {
-        allowNull: false,
-        type: Sequelize.DATE
-      },
-      patronId: {
-        allowNull: false,
-        type: Sequelize.INTEGER
-      },      
-      returned: {
-        allowNull: false,
-        type: Sequelize.BOOLEAN,
-        defaultValue: false
-      }
-    });
-  }
+export function up(queryInterface, Sequelize) {
```

Comment:
Read up this: https://sequelize.readthedocs.io/en/latest/docs/migrations/#addcolumntablenameoroptions-attributename-datatypeoroptions-options

Since you are making changes to an existing table, a new migration file should be created to add the new columns instead of adding to the existing file. This ensures the migration call gets called and the changes reflects on the DB without clearing everything again